docs/book/src/component-config-tutorial/testdata/project/main.go

```diff
@@ -62,10 +62,9 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
 	var err error
-	ctrlConfig := configv2.ProjectConfig{}
 	options := ctrl.Options{Scheme: scheme}
 	if configFile != "" {
-		options, err = options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(&ctrlConfig))
+		options, err = options.AndFrom(ctrl.ConfigFile().AtPath(configFile))
```

Comment:
The source/code here, the example from the tutorial was changed.
Could you please revert?